Page/Wishlist/wishlist.css

```diff
@@ -0,0 +1,19 @@
+.wishlist-container-box{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
can use the common properties in utility class

Reply:
updated